.vscode/settings.json

```diff
@@ -9,6 +9,9 @@
 	"files.eol": "\n",
 	"files.trimTrailingWhitespace": true,
 
+	"task.allowAutomaticTasks": "on",
```

Comment:
Is the` settings.json` change intended?

Reply:
Yes this is intended to auto start the compiler / watch task